sphinx/texinputs/sphinx.sty

```diff
@@ -8,26 +8,15 @@
 \NeedsTeXFormat{LaTeX2e}[1995/12/01]
 \ProvidesPackage{sphinx}[2017/02/18 v1.6 LaTeX package (Sphinx markup)]
 
-% this is the \ltx@ifundefined of ltxcmds.sty, which is loaded by
-% kvoptions (and later by hyperref), but the first release of
-% ltxcmds.sty as in TL2009/Debian has wrong definition.
-\newcommand{\spx@ifundefined}[1]{%
-    \ifcsname #1\endcsname
-      \expandafter\ifx\csname #1\endcsname\relax
-        \expandafter\expandafter\expandafter\@firstoftwo
-      \else
-        \expandafter\expandafter\expandafter\@secondoftwo
-      \fi
-    \else
-      \expandafter\@firstoftwo
-    \fi
-}
+% \spx@ifundefined was defined here at time Sphinx supported TL2009/Debian
+% as corrected version of \ltx@ifundefined. Not needed anymore as original ok.
```

Comment:
thanks for quick review @tk0miya I agree and will modify comment. By the way package ltxcmds is anyhow a dependency of many packages like kvoptions or hyperref etc...